src/logger.js

```diff
@@ -63,11 +63,9 @@ class Logger {
 
   emit(type: string, args: LoggingArgs): void {
     const fn = console[type];
-    if (fn == null) {
-      throw new Error(`Unknown \`console\` function or unavailable: ${type}`);
```

Comment:
Rather than removing this entirely, perhaps we should log to `console.error` instead? Otherwise I think the log message will just be lost, which could lead to confusion.

Reply:
See latest commit.